core/dist/css/decanter-grid.css

```diff
@@ -674,498 +674,1590 @@ body {
+        -ms-flex: 0 0 calc(75% - 12px);
+            flex: 0 0 calc(75% - 12px);
+    max-width: calc(75% - 12px); } }
+
+@media only screen and (min-width: 1200px) {
+  .flex-xl-10-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(83.3333333333% - 6.6666666667px);
+            flex: 0 0 calc(83.3333333333% - 6.6666666667px);
+    max-width: calc(83.3333333333% - 6.6666666667px); } }
+
+@media only screen and (min-width: 1500px) {
+  .flex-xl-10-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(83.3333333333% - 8px);
+            flex: 0 0 calc(83.3333333333% - 8px);
+    max-width: calc(83.3333333333% - 8px); } }
+
+@media only screen and (min-width: 1200px) {
+  .flex-xl-11-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(91.6666666667% - 3.3333333333px);
+            flex: 0 0 calc(91.6666666667% - 3.3333333333px);
+    max-width: calc(91.6666666667% - 3.3333333333px); } }
+
+@media only screen and (min-width: 1500px) {
+  .flex-xl-11-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(91.6666666667% - 4px);
+            flex: 0 0 calc(91.6666666667% - 4px);
+    max-width: calc(91.6666666667% - 4px); } }
+
+@media only screen and (min-width: 1200px) {
+  .flex-xl-12-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(100% - 0px);
+            flex: 0 0 calc(100% - 0px);
+    max-width: calc(100% - 0px); } }
+
+@media only screen and (min-width: 1500px) {
+  .flex-xl-12-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(100% - 0px);
+            flex: 0 0 calc(100% - 0px);
+    max-width: calc(100% - 0px); } }
+
+@media only screen and (min-width: 576px) {
+  .flex-2xl-1-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(8.3333333333% - 18.3333333333px);
+            flex: 0 0 calc(8.3333333333% - 18.3333333333px);
+    max-width: calc(8.3333333333% - 18.3333333333px); } }
+
+@media only screen and (min-width: 768px) {
+  .flex-2xl-1-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(8.3333333333% - 18.3333333333px);
+            flex: 0 0 calc(8.3333333333% - 18.3333333333px);
+    max-width: calc(8.3333333333% - 18.3333333333px); } }
+
+@media only screen and (min-width: 992px) {
+  .flex-2xl-1-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(8.3333333333% - 33px);
+            flex: 0 0 calc(8.3333333333% - 33px);
+    max-width: calc(8.3333333333% - 33px); } }
+
+@media only screen and (min-width: 1200px) {
+  .flex-2xl-1-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(8.3333333333% - 36.6666666667px);
+            flex: 0 0 calc(8.3333333333% - 36.6666666667px);
+    max-width: calc(8.3333333333% - 36.6666666667px); } }
+
+@media only screen and (min-width: 1500px) {
+  .flex-2xl-1-of-12 {
+    -webkit-box-flex: 0;
+        -ms-flex: 0 0 calc(8.3333333333% - 44px);
+            flex: 0 0 calc(8.3333333333% - 44px);
+    max-width: calc(8.3333333333% - 44px); } }
+
+@media only screen and (min-width: 576px) {
```

Comment:
It looks like for example, the `.flex-xl-x-of-12` classes start with media query for the `xl` breakpoint (which is right), but the `.flex-2xl-x-of-12` classes start with media query for the `xs` breakpoint? Seems like they should just start at `2xl` breakpoint?
![Screen Shot 2019-06-12 at 1 59 49 AM](https://user-images.githubusercontent.com/42749717/59338345-0a3a2f80-8cb7-11e9-99e6-a9857da218f4.png)

Reply:
Yup, something's weird with the `2xl `and up classes 👀  Screenshot is at `sm` breakpoint.
![Screen Shot 2019-06-12 at 2 14 53 AM](https://user-images.githubusercontent.com/42749717/59338879-0b1f9100-8cb8-11e9-9758-63292bdb7390.png)

Reply:
This one works great - thanks!